support-figma/extended-layout-plugin/ui.html

```diff
@@ -156,6 +156,12 @@
           <label for="barVerticalSwitch" class="switch__label" style="font-weight:normal">Vertical</label>
         </div>
       </div>
+      <div style="display: flex">
+        <div class="switch">
+          <input id="barMoveOnlySwitch" type="checkbox" class="switch__toggle">
+          <label for="barMoveOnlySwitch" class="switch__label" style="font-weight:normal">Move Only (Scrubber)</label>
```

Comment:
"Move Only" sounds a bit funny, but "Translate" sounds quite technical. I guess we can figure out the right way to name this once Aubrey gets into designing the plugin panel (but then we will need to update JS, Rust, and Kotlin...).

Reply:
I really struggled to figure out what to call this. I was thinking of this as a tick mark, so maybe `tickMode` but that didn't sound right.

Reply:
What about reframing it from a boolean to an option named "progressIndicatorMode" or "indicatorMode", with the options being "expanding" or "positionOnly".

Or allow both options to exist at the same time? Have an "expanding" element and a "position" element, and just render both if frames are given for both.

Reply:
Heh, my feedback on the first round was to only allow one option at a time because it was hard for me to think of valid use cases for combinations, and because it multiplies the validation effort.

Fundamentally, we're either "transforming" (rotate or translate) or "resizing" (along an arc, or along a rect/frame's axis); that's a very low level description, but perhaps that's OK given all of the different ways of composing the drawing capabilities with this feature result in many use cases (progress, dials, gauges, indicators...) getting tackled.

@miriamvu wdyt?

Reply:
It would be harder to support having both since this is on one frame or one rectangle vector.

Reply:
I agree that we should allow one option at a time to keep things simpler and easier to understand/use.  As for terminology, it seems that the there are three possible operations:  translation, rotation, and scaling (Figma uses this term too for expanding and shrinking an item:  https://help.figma.com/hc/en-us/articles/360040451453-Resize-layers-with-the-scale-tool).   Hope this helps!